lib/core/src/server/manager/manager-webpack.config.js

```diff
@@ -42,6 +45,17 @@ export default ({
     encoding: 'utf8',
   });
 
+  if (legacy) {
+    logger.info("=> Transpiling some node_modules, you're brave..");
+  }
+
+  // Don't know why webpack resolves esm modules...
+  const resolveExtension = legacy
```

Comment:
Any idea why I had to do this ? https://github.com/webpack/webpack/issues/5756

overlayscrollbar is the source of this hack..